olympus.sh

```diff
@@ -16,17 +16,19 @@ if [ -f "olympus.new.love" ]; then
     mv "olympus.new.love" "olympus.love"
 fi
 
-if [ "$UNAME" == "Darwin" ]; then
-	export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:$(pwd)"
+if [ "$(uname)" = "Darwin" ]; then
```

Comment:
I've adapted this from the old MonoKickstart wrapper script. I'm not entirely sure if `uname` behaves the same on macOS as I don't have a Mac to test things on and always go by previous working practices if possible.